tests/zero_bounce_test_case.py

```diff
@@ -246,3 +246,61 @@ def test_delete_file_valid(self):
         self.assertEqual(response.message, "File Deleted")
         self.assertEqual(response.file_id, "5e87c21f-45b2-4803-8daf-307f29fa7340")
         self.assertEqual(response.file_name, "emails.txt")
+
+    def test_find_email_status_invalid(self):
+        self.requests_mock.get.return_value = mock_response = MockResponse({
+            "email": "",
+            "domain": "example.com",
+            "format": "unknown",
+            "status": "invalid",
+            "sub_status": "no_dns_entries",
+            "confidence": "undetermined",
+            "did_you_mean": "",
+            "failure_reason": "",
+            "other_domain_formats": []
+        })
+
+        response = self.zero_bounce_client.find_email(
+            "example.com", "John", "", "Doe"
+        )
+        for field, expected_value in mock_response.json().items():
```

Comment:
not a good way to check. status, sub_status (at least, if not confidence and failure_reason as well) should be objects so you won't be able to compare the string with the object

Reply:
will change, after adding enums